git.adoc

```diff
@@ -10,35 +10,33 @@ Sharing some git conveniences `~/.bashrc` aliases and scripts.
 +
 [source,text]
 ----
-#!/usr/bin/bash
+#!/usr/bin/env bash
 
-cd ~/openshift-docs
-if git rev-parse --is-inside-work-tree > /dev/null 2>&1; then
-  echo "Proceeding with rebase."
-else
-  echo "This is not a git repository. Please check your path syntax."
-  exit
+OPENSHIFT_MAIN_BRANCH="main"
+OPENSHIFT_UPSTREAM_REMOTE="upstream"
+OPENSHIFT_PERSONAL_REMOTE="origin"
+
+set -e
```

Comment:
Makes the script fail upon any error a command throws (doesn't apply to statements in ifs). Useful when an invalid or empty branch name is entered etc. You could get very unexpected behaviour otherwise.